pyproject.toml

```diff
@@ -7,7 +7,7 @@ authors = ["Leon Handreke <leonh@ndreke.de>"]
 [tool.poetry.dependencies]
 python = "^3.10"
 Django = "~3.2.16"
-django-ldapdb = "^1.5.1"
+django-ldapdb = {version="^1.5.1", markers="sys_platform=='linux'"}
```

Comment:
Could we make this linux or macos?

Reply:
Added it, maybe someone with a mac can confirm it's working?

Reply:
I checked on my work's macbook, it worked.